devtool

```diff
@@ -167,6 +167,15 @@ install_poetry() {
 install_deps() {
     install_poetry
     poetry install
+    install_boto_3_stubs_package
+}
+
+install_boto_3_stubs_package() {
+  # For some reason, simply adding boto3-stubs to poetry.lock
+  # isn't sufficient. We need to run the following commands
+  # for the mypy type annotations to work.
```

Comment:
Is mypy the only problem? Can't we just ignore mypy?

Reply:
I figured out how to install the required package (`mypy-boto3-bedrock`) thorugh poetry, so we no longer need to run these commands manually.